### ISSUE TITLE
Redundant empty line also on object initializer syntax

### ISSUE THREAD

**Issue by MSeifert04** [JosefPihrt/Roslynator#415]
For example empty lines at the beginning or end of the init-block:

```
    class Test
    {
        public int A { get; set; }
        public int B { get; set; }
    }

    class Test2
    {
        public Test2()
        {
            var a = new Test
            {

                A = 1,
                B = 2

            };
        }
    }
```

The line before `A = 1` and the line after `B = 2` could trigger the "remove empty line" warning.

For example if it were an `if` these are triggered:

```
            if (true)
            {

                var A = 1;
                var B = 2;

            };
```

There it emits the warnings for the first and last line (empty) of the block.
